.github/workflows/on-merge-branch-changeset-release-main.yml

```diff
@@ -12,7 +12,7 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  install_dependencies:
+  install-dependencies:
```

Comment:
GitHub's documentation uses both snake and kebab case. But it appears to use exclusively use kebab case for properties in practice. So I switched ours to kebab to match.